### PR TITLE
feat: notify query execute successfully when app is not focused.

### PR DESCRIPTION
**Pull request by invisal**
https://github.com/beekeeper-studio/beekeeper-studio/issues/1435

We will push notification only when the app is not in focused. We can make the notification look better if we have some images.

**For Windows Screenshot**
image

**For Mac Screenshot**
image

**How to test**
- Connect to MySQL server.
- Run this query `SELECT SLEEP(10);`
- Open other app besides Beekeeper and wait
- The notification will show after 10 seconds when it is complete.

### CODE REVIEW

**Comment by rathboma** [comment]
This is awesome! Does this create a native notification on different OSes, or a weird electron-looking one?

**Comment by invisal** [comment]
I believe it should be native. In Window, it appears in the bottom right side. I will try to test on Mac to see how it looks like, but because my Mac is M1, it seem to have some problem running the project. Let me try again and provide you with Mac screenshot.

**Comment by invisal** [comment]
@rathboma I have updated the screenshot for Mac. The thing that need to do is wording. My english is terrible and maybe you can give suggestion.

**Review by rathboma — APPROVED** [review]
